tests/test_plot_widget.py

```diff
@@ -0,0 +1,30 @@
+"""
+Tests for plot_widget.py
+"""
+import pytest
+from plot_widget import PlotWidget
+
+
+@pytest.fixture
+def test_app(qtbot):
+    """
+    Sets up widget for testing
+    """
+    test_app = PlotWidget()
+    qtbot.addWidget(test_app)
+
+    return test_app
+
+
+def test_plot_dcm(test_app):
+    """
+    Tests plot_widget test_plot_dcm function
+    """
+    assert test_app.plot_dcm("test files/DICOM-RT-01/CT_3_Hashed.dcm")
```

Comment:
Are we sure this is correct? another way of testing could be running through the whole process of clicking import and selecting the file, however that could be tricky

Reply:
This is a correct way of testing since it tests the process of plotting, it does not test the ui such as clicking file > clicking open dicom file > select file > then plot_dcm, it skips the ui testing part and only the functionality. 

The ui testing will be hard at first, not too sure how to do the file dialog automatically but there is documentation on testing menus

Reply:
what is it actually asserting though, that an error isn't thrown?

Reply:
plot_dcm returns a bool, this bool is determined by whether or not the axis is 'on' or 'off'. Not sure of any other way to test whether something has been plotted or not

Reply:
How would you test the same function, but through the ui? It still needs to return something so it can be asserted, although  prob could just assert if an error has been thrown

Reply:
ah okay it seems like i didnt get the functionality of self.axes.axis() != (0.0, 1.0, 0.0, 1.0) all good